hooks/createNotificationService.js

```diff
@@ -0,0 +1,5 @@
+var config = {};
+
+function createNotificationService(config){
+
+};
```

Comment:
Unneccesarry semi colon

Reply:
Ok, I will remove it.

But it is a good javascript practice to have semi colon in method.